department-of-veterans-affairs/va.gov-cms
# VACMS-5201: Add vc types to preview exclusion list.

## Description
See #5201 

## Testing done
Visual / behat

## QA steps
Visually verify that existing node view pages for the following types do not have the `Preview` button:
'vet_center_cap',
'vet_center_facility_health_servi',
'vet_center_mobile_vet_center',
'vet_center_outstation',

### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
